common/presenters/move-to-meta-list-component.js

```diff
@@ -105,7 +118,7 @@ function moveToMetaListComponent(move, { updateUrls = {} } = {}) {
         text: i18n.t('fields::date_type.label'),
       },
       value: {
-        text: filters.formatDateWithRelativeDay(date),
+        text: journeysSummary.map(({ date }) => date).join(' to '),
```

Comment:
It looks like this will state `Monday 28 Mar 2022 to Tuesday 29 Mar 2022 to Wednesday 30 Mar 2022 to... (etc etc)` would and work better?

Reply:
I am pretty neutral tbh, I am going off the content designs. As my content/design outlook is pretty bad haha

Reply:
Although this code is generic, I'm not aware of any situation where they'll be more than 2 journeys anyway, so I don't think this will be a big deal.